confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConstant.java

```diff
@@ -20,11 +20,31 @@
 
 public class ConfigNodeConstant {
 
+  // when running the program in IDE, we can not get the version info using
+  // getImplementationVersion()
+  public static final String VERSION =
+      ConfigNodeConstant.class.getPackage().getImplementationVersion() != null
+          ? ConfigNodeConstant.class.getPackage().getImplementationVersion()
+          : "UNKNOWN";
+
+  public static final String GLOBAL_NAME = "IoTDB ConfigNode";
+  public static final String CONFIGNODE_CONF = "CONFIGNODE_CONF";
+  public static final String CONFIGNODE_HOME = "CONFIGNODE_HOME";
+
+  public static final String ENV_FILE_NAME = "confignode-env";
+  public static final String CONF_NAME = "iotdb-confignode.properties";
+  public static final String SPECIAL_CONF_NAME = "iotdb-confignode-special.properties";
+
+  public static final String CONFIGNODE_PACKAGE = "org.apache.iotdb.confignode.service";
+  public static final String JMX_TYPE = "type";
+  public static final String CONFIGNODE_JMX_PORT = "confignode.jmx.port";
+
+  public static final String DATA_DIR = "data";
```

Comment:
Add the config_data_dir parameter to the configuration item to support mounting of multiple disks.
 see :"DirectoryManager" in server